NewHorizons/Builder/Volumes/VolumesBuildManager.cs

```diff
@@ -49,6 +50,34 @@ public static void Make(GameObject go, Sector sector, OWRigidbody planetBody, Pl
                     HazardVolumeBuilder.Make(go, sector, planetBody, hazardVolume, mod);
                 }
             }
+            if (config.Volumes.mapRestrictionVolumes != null)
+            {
+                foreach (var mapRestrictionVolume in config.Volumes.mapRestrictionVolumes)
+                {
+                    VolumeBuilder.Make<MapRestrictionVolume>(go, sector, mapRestrictionVolume);
+                }
+            }
+            if (config.Volumes.interferenceVolumes != null)
+            {
+                foreach (var interferenceVolume in config.Volumes.interferenceVolumes)
+                {
+                    VolumeBuilder.Make<Components.InterferenceVolume>(go, sector, interferenceVolume);
+                }
+            }
+            if (config.Volumes.reverbVolumes != null)
+            {
+                foreach (var reverbVolume in config.Volumes.reverbVolumes)
+                {
+                    VolumeBuilder.Make<ReverbTriggerVolume>(go, sector, reverbVolume);
+                }
+            }
+            if (config.Volumes.insulatingVolumes != null)
+            {
+                foreach (var insulatingVolume in config.Volumes.insulatingVolumes)
+                {
+                    VolumeBuilder.Make<InsulatingVolume>(go, sector, insulatingVolume);
```

Comment:
Do you have a usecase in mind for these? Maybe if we could parent volumes to things like we do for props and such.

Reply:
The only thing I can think it would be used for is when an addon creator wants an electricity hazard volume to not affect you in a certain area.

Reply:
I'll add the parent path to volumes.